.github/workflows/changelog-update.yml

```
@@ -0,0 +1,15 @@
+name: "Changelog update"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+
+jobs:
+  # Enforces the update of a changelog file on every pull request
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dangoslen/changelog-enforcer@v2
+        with:
+          changeLogPath: 'CHANGELOG.rst'
+          skipLabels: 'Skip-Changelog'
```

A file should always end with an empty new line
```suggestion
          skipLabels: 'Skip-Changelog'
          
```